web/samples_index/lib/src/carousel.dart

```diff
@@ -0,0 +1,210 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file
+
+import 'dart:html';
+
+class Carousel {
+  final bool withArrowKeyControl;
+
+  final container = querySelector('.slider-container');
+  var slides = querySelectorAll('.slider-single');
```

Comment:
nit: give these a type declaration:

```suggestion
  final Element container = querySelector('.slider-container');
  List<Element> slides = querySelectorAll('.slider-single');
```